docs_app/tools/transforms/angular-api-package/index.js

```diff
@@ -162,17 +165,33 @@ module.exports = new Package('angular-api', [basePackage, typeScriptPackage])
       pathTemplate: '${moduleDoc.moduleFolder}/${name}',
       outputPathTemplate: '${moduleDoc.moduleFolder}/${name}.json',
     });
+    // Override path for symbols that would otherwise collide with other names (index/observable vs. index/Observable)
+    // on case insensitive filesystems (eg. OS X).
+    computePathsProcessor.pathTemplates.push({
+      docTypes: ['const'],
+      getPath: (doc) => {
```

Comment:
@petebacondarwin suggested to always use the type in the url and therefore be more generic. And to be honest I would really like this  :D